lib/_tls_wrap.js

```diff
@@ -114,6 +114,12 @@ function requestOCSP(self, hello, ctx, cb) {
 
   if (!ctx)
     ctx = self.server._sharedCreds;
+
+  // Running on non-TLS server
```

Comment:
This comment confused me when I read it, and I only understood when reading the unit test and PR history. Can I suggest:

> TLS socket is using a net.Server, instead of a tls.TLSServer, so some TLS properties will not be present.